src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount;
+  const priceForDays = 40;
+  const saleDiscountUp7Days = 20;
+  const saleDiscountAfter7Days = 50;
+
+  if (days < 3) {
+    amount = (days * priceForDays);
```

Comment:
Don't repeat yourself: create variable for days * priceForDays value

Reply:
You can also set one of values by default, one of this if statements is redundant here